src/assets/data/team/team-members.json

```diff
@@ -0,0 +1,87 @@
+[
```

Comment:
We need to add everyone's LinkedIn information

Reply:
Yeah, is there somewhere I can get all that?